core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -115,6 +116,7 @@ import org.slf4j.Logger
     statement.execute().asFuture().flatMap { result =>
       val resultPublisher: Publisher[A] =
         result.map((row, _) => mapRow(row))
+      // note: it's already indexed seq (vector) so toindexedSeq is essentially no op, not copy
```

Comment:
it's a "cast" because I preferred to have IndexedSeq in the result type rather than `Seq` from the `Sink.seq`